wolfssl/tests/ping_pong.rs

```diff
@@ -193,51 +203,58 @@ async fn server<S: SockIO>(sock: S, protocol: Protocol) {
     println!("[Server] Finished");
 }
 
-#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_3 => panics; "client_1.2_server_1.3")]
-#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_2; "client_1.2_server_1.2")]
-#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServer; "client_1.2_server")]
-#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServerV1_3; "client_1.3_server_1.3")]
-#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServerV1_2 => panics; "client_1.3_server_1.2")]
-#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServer; "client_1.3_server")]
-#[test_case(Protocol::DtlsClient, Protocol::DtlsServerV1_3; "client_server_1.3")]
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_3, ProtocolVersion::Unknown => panics; "client_1.2_server_1.3")]
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_2, ProtocolVersion::DtlsV1_2; "client_1.2_server_1.2")]
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServer, ProtocolVersion::DtlsV1_2; "client_1.2_server")]
+#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServerV1_3, ProtocolVersion::DtlsV1_3; "client_1.3_server_1.3")]
+#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServerV1_2, ProtocolVersion::Unknown => panics; "client_1.3_server_1.2")]
+#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServer, ProtocolVersion::DtlsV1_3; "client_1.3_server")]
+#[test_case(Protocol::DtlsClient, Protocol::DtlsServerV1_3, ProtocolVersion::DtlsV1_3; "client_server_1.3")]
```

Comment:
Shall we consider adding whitespace after each `,` and the `;` to make these more tabular (=> readable)?

Reply:
I do not want to go down this route of hand editing the source code as tabular format.
IMHO it might be useful for readme and documentation, but not for test cases.
I prefer to follow what `cargo fmt` provides

Reply:
`cargo fmt` doesn't touch the contents of `test_case(...)` so you won't be fighting it here.

IME representing test cases as tables is a really helpful technique and it works nicely in the source e.g. https://github.com/xvpn/password_manager/blob/master/pmcore/src/tests/unlock.rs#L529

Reply:
IMHO its not worth the time to manually beautify the `test` code unless a tool does it.
For example:
https://github.com/xvpn/lightway/blob/main/lightway-core/src/wire.rs#L491

We have lot of test cases here, which will be clumsy. We can spend of time making it readable.
But it may not be worth it.

Reply:
IMHO it is, in general, absolutely worth it, a tabular form makes it much easier to see if all cases are covered -- I made this comment because I was doing exactly that.

There are exceptions and I think the link you gave is to one of them, aliging things there would result in unmanageable line lengths.